modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1980,11 +1992,38 @@ struct Mapper {
         return u;
     }
 };
+}  // namespace linear
+
+namespace linear32f {
+struct Mapper {
```

Comment:
looks familiar https://github.com/opencv/opencv/pull/21678/files#diff-7f0bda22bce310481b13ffb6c7114b9c957d864de2b95530dbe994b83c577d5bR1996

Is it kind of duplication?

Reply:
This patch is the successor of  the `GAPI Fluid: Resize F32C1 scalar version. `#21678. That is, it contains that patch plus the SIMD add-on.